frontend/scripts/react-components/data-portal/data-portal-form.component.jsx

```diff
@@ -114,8 +114,8 @@ class DataPortalForm extends Component {
                 We&apos;d love to hear about how you use the data and how we could improve it.
                 Please fill in details below and click on &apos;submit&apos;:
               </p>
-              <label htmlFor="name">
-                Name:
+              <label htmlFor="name" tx-content="exclude">
```

Comment:
could we move the input elements outside of the label element? this way, I think we are flagging the input as exclude, which I assume will result in the placeholder being ignored by transifex

Reply:
Alright, I managed to fix that issue in another way. The problem is that Transifex is re-adding input nodes on itself which is not friendly at all with React VDOM, but I think I have a working version, just need a few more minutes to test it and I will rebase that branch.